spec/forms/csv_upload_form_spec.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/file_upload_helpers'
+
+RSpec.describe CsvUploadForm, type: :form do
+  describe '#save' do
+    context 'given a VALID csv file' do
```

Comment:
```suggestion
    context 'given a VALID CSV file' do
```

Reply:
Fixed in ab157a4212523e8ffed59a8096fa4e5377ae8a04